recipes-openxt/initrdscripts/initramfs-xenclient/init.sh

```diff
@@ -169,10 +169,10 @@ tpm_setup() {
     insmod /lib/modules/$(uname -r)/kernel/drivers/char/tpm/tpm.ko
     insmod /lib/modules/$(uname -r)/kernel/drivers/char/tpm/tpm_tis.ko
     mknod /dev/tpm0 c 10 224
-    echo -n "initramfs measuring $ROOT_DEVICE: "
```

Comment:
Running sha1sum on the whole root partition can be a lengthy process, especially on spinning hard drives. We should let the user/dev know what's happening.
If you don't like the `-n`, maybe just do an `echo Measuring rootfs...` before, and `echo Done. Hash = $s` after or something.